app/src/main/java/org/projectbuendia/client/ui/chart/ChartRenderer.java

```diff
@@ -87,11 +91,33 @@ public void render(Chart chart, Map<String, Obs> latestObservations,
 
         mView.getSettings().setJavaScriptEnabled(true);
         mView.addJavascriptInterface(controllerInterface, "controller");
-        mView.setWebChromeClient(new WebChromeClient());
+
+        mView.setWebChromeClient(new WebChromeClient(){
+            @Override
+            public boolean onJsAlert(WebView view, String url, String message, final android.webkit.JsResult result)
+            {
+                new AlertDialog.Builder(mView.getContext())
+                        .setTitle(mView.getContext().getString(R.string.observation))
```

Comment:
If you set the title as observation, every time a alert() is called in the chart it will have the title "Observation" even if it is not related. Just remove setTitle() and you will have a alert without title, like Firefox:
![image](https://cloud.githubusercontent.com/assets/1710823/10933856/b3a97542-82c2-11e5-81c5-a14fdbd5c56a.png)

![device-2015-11-04-070943](https://cloud.githubusercontent.com/assets/1710823/10933901/21805a22-82c3-11e5-8469-7881476da641.png)